src/ideascale/mod.rs

```diff
@@ -85,7 +85,7 @@ pub async fn fetch_all(
         funnels,
         fund: funds
             .into_iter()
-            .find(|f| f.name.as_ref().contains(&format!("Fund{}", fund)))
+            .find(|f| f.name.as_ref().contains(&format!("Fund {}", fund)))
```

Comment:
Why oh why?

Reply:
they added space in name :) But agree i could do regex